src/fiatExchanges/CoinbasePaymentSection.test.tsx

```diff
@@ -47,6 +48,7 @@ describe('CoinbasePaymentSection', () => {
     })
   })
   it('shows nothing if coinbase is restricted and feature flag is false', async () => {
+    mocked(readOnceFromFirebase).mockResolvedValue(FAKE_APP_ID)
```

Comment:
I'm confused, why are these mocked firebase reads necessary now? Is this a new requirement that's introduced by your analytics changes? or is it an unrelated fix you're bundling in with this PR? (if it's the latter, please split it out to a separate PR)